package.json

```diff
@@ -5,8 +5,9 @@
     "install:all": "npm install && cd server && npm install && cd ../client && npm install",
     "start": "concurrently \"npm run start:server\" \"npm run start:client\"",
     "start:docker": "cross-env NODE_ENV=docker npm start",
-    "start:server": "cd server && nodemon server",
+    "start:server": "cd server && npm run start",
```

Comment:
please explain me that change

Reply:
We can't just use nodemon because of TypeScript so command npm `run start`
1. convert our .ts files to .js files and put them on ./build directory
2. run ./build/server via node 
and this is case for production.
And if you want to run server in development mode you need to run "dev" in server directory. 
Thanks to ts-node-dev you have autorestarts on .ts file change.

Reply:
But now I see start:client run react in development mode. So I'll change start:server to run server in development mode also. We need set command like start:production or something like that to build our server and client.